### PR TITLE
Merge staging to production

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.ts [hunk 1]
```diff
@@ -366,6 +366,7 @@ export class ExpenseDetailsComponent implements OnInit, OnChanges {
                 this.signatureSrc = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + output.file.response.body.uniqueName;
                 let img = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + output.file.response.body.uniqueName;
                 this.DownloadAttachedImgResponse.push(output.file.response.body);
+                this.imgAttachedFileName = output.file.response.body.name;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.ts:369]: Why this is redundant?

Comment (rohit-walkover) [reply apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.ts:369]: API need extra params file name As ledger support only single attached file

File: apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.ts [hunk 2]
```diff
@@ -517,4 +527,57 @@ export class InvoicePreviewDetailsComponent implements OnInit, OnChanges, AfterV
...
+            this.salesService.getAllAdvanceReceiptVoucher(requestObject).subscribe(res => {
+                if (res && res.status === 'success') {
+                    if (res.body && res.body.length) {
+                        this.isAccountHaveAdvanceReceipts = true;
+                    } else {
+                        this.isAccountHaveAdvanceReceipts = false;
+                    }
+                    this.detectChanges();
+                } else {
+                    this.isAccountHaveAdvanceReceipts = false;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.ts:578]: Add detect changes here also or after if/else

File: apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html [hunk 3]
```diff
@@ -18,11 +18,11 @@ <h3 class="aside-title">Payment</h3>
                             <label>Beneficiary Name</label>
                             <input class="form-control" [value]="accountDetails?.accountBankDetails[0]?.beneficiaryName" type="text" disabled />
                         </div>
-                        <div class="col-sm-6 col-xs-12" *ngIf="countryCode ==='AE'">
+                        <div class="col-sm-6 col-xs-12" [ngClass]="{'mrT1': countryCode ==='AE'}">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html:21]: Use class mt-1

File: apps/web-giddh/src/app/create-advance-receipt/create-advance-receipt.component.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        <div class="col-sm-8 font-14">
+            <div class="row">
+                <label class="col-md-2 pt-05">Customer Name<span class="text-danger">*</span></label>
+                <div class="col-md-4">
+                    <div class="form-group">
+                        <sh-select class="text-left dropdown-arrow" name="adjustmentVoucher" placeholder="Customer Name" [notFoundLink]="false"
+                        [multiple]="false" [ItemHeight]="33">
+                            <ng-template #optionTemplate1 let-option="option">
+                                <a href="javascript:void(0)" class="list-item">
+                                    Advance Receipt
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/create-advance-receipt/create-advance-receipt.component.html:12]: Why we are using static content?
 ravinder ji created JSON for static content

File: apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.ts [hunk 5]
```diff
@@ -32,6 +32,8 @@ export class ExportDaybookComponent implements OnInit, OnDestroy {
                 this.emailTypeSelected = isAdmin ? 'admin-detailed' : 'view-detailed';
                 this.emailTypeMini = isAdmin ? 'admin-condensed' : 'view-condensed';
                 this.emailTypeDetail = isAdmin ? 'admin-detailed' : 'view-detailed';
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.ts:35]: Remove blank space

File: apps/web-giddh/src/app/inventory/inventory.component.html [hunk 6]
```diff
@@ -12,12 +12,15 @@
                     <section class="rightBar">
                         <!-- uiView: inventory-detail -->
                         <section class="container-fluid  clearfix pdT2">
-
+                            <div *ngIf="activeView" class="d-flex mrB1 branch-warehouse-filter">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/inventory/inventory.component.html:15]: mrB1 Use class name mb-1

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.html [hunk 7]
```diff
@@ -94,21 +94,18 @@
...
 
+    <div class="table-responsive">
     <table class="table basic table-bordered pending-tabel onMobileView mrT2">
-
         <thead class="t-h-bg1">
             <tr>
-
                 <th style="width:4%" *ngIf="selectedLedgerItems.length > 0">
 
                     <div class="mh18_img" style="justify-content: center;display: flex">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/generate/invoice.generate.component.html:105]: Remove inline css

**Pull request merged**